scripts/code_style.py

```diff
@@ -174,22 +182,23 @@ def main() -> int:
     parser.add_argument('-f', '--fix', action='store_true',
                         help=('modify source files to fix the code style '
                               '(default: print diff, do not modify files)'))
-    # --files is almost useless: it only matters if there are no files
-    # ('code_style.py' without arguments checks all files known to Git,
-    # 'code_style.py --files' does nothing). In particular,
-    # 'code_style.py --fix --files ...' is intended as a stable ("porcelain")
-    # way to restyle a possibly empty set of files.
-    parser.add_argument('--files', action='store_true',
-                        help='only check the specified files (default with non-option arguments)')
+    parser.add_argument('--subset', action='store_true',
+                        help=('check a subset of the files known to git '
+                              '(default: check all files passed as arguments, '
+                              'known to git or not)'))
     parser.add_argument('operands', nargs='*', metavar='FILE',
                         help='files to check (if none: check files that are known to git)')
```

Comment:
Partly preexisting:
```suggestion
                        help='files to check (if none and no --subset: check files that are known to git)')
```
Which is a weird interface, see my comment above.